molecule/default/molecule.yml

```diff
@@ -3,9 +3,7 @@
 dependency:
   name: galaxy
 driver:
-  name: docker
-lint:
-  enabled: false
+  name: ${LSR_MOLECULE_DRIVER:-docker}
```

Comment:
what is the `LSR_MOLECULE_DRIVER` stands for? Is there any link available for me to get reference?

Reply:
The variable id described in https://github.com/linux-system-roles/tox-lsr/#using-setenv-and-environment-variables.

LSR stands for Linux System Roles. :)

Reply:
> what is the `LSR_MOLECULE_DRIVER` stands for? Is there any link available for me to get reference?

so you can do something like `LSR_MOLECULE_DRIVER=podman tox -e molecule` to use `podman` instead of the default `docker`